lib/lotus/cli.rb

```diff
@@ -83,6 +83,7 @@ def new(name = nil)
     method_option :application_base_url, desc: 'application base url',                                      type: :string
     method_option :path,                 desc: 'applications path',                                         type: :string
     method_option :url,                  desc: 'relative URL for action',                                   type: :string
+    method_option :method,               desc: "HTTP method for action. Upper/lower case is ignored. Must be one of GET, POST, PUT, PATCH, DELETE, HEAD, OPTIONS, TRACE.", type: :string, default: 'GET'
```

Comment:
It would force me to require the class. Thought that might be not wanted for a description (it's loaded all the time then, even for other commands). Should be easier/cleaner once these commands have been refactored like the db commands.
What do you think? Require it seems fine to me.